dnsmasq.json

```diff
@@ -3,6 +3,10 @@
 	"plugin_schema": "2",
 	"release": "11.2-RELEASE",
 	"artifact": "https://github.com/timsavage/iocage-plugin-dnsmasq.git",
+        "properties": {
+            "nat": 1,
+            "nat_forwards": "tcp(53:53)"
```

Comment:
I am not sure how `dnsmasq` operates, but is it necessary that it uses host's `53` port ? Middlewared will not start the jail because it doesn't allow changes to ports less then `1025`